h/views/api_flags.py

```diff
@@ -16,8 +20,18 @@
             effective_principals=security.Authenticated)
 def create(context, request):
     annotation = _fetch_annotation(context, request)
+
     svc = request.find_service(name='flag')
     svc.create(request.authenticated_user, annotation)
+
+    group_service = request.find_service(IGroupService)
+    group = group_service.find(annotation.groupid)
+
+    incontext_link = links.incontext_link(request, annotation)
+
+    send_params = flag_notification.generate(request, group.creator.email, incontext_link)
```

Comment:
In case the annotation was made in the `__world__` group then the group service returns a [`WorldGroup`](https://github.com/hypothesis/h/blob/master/h/groups/util.py#L8) instance, which doesn't have a creator (it doesn't even have a `creator` property at the moment), which will then produce an exception here.

For now, maybe you can add a `creator` property in the `WorldGroup` class which just returns `None`, and then here check if `group.creator is not None`, and otherwise don't send an email. We really need to move the `__world__` group into the database to avoid these things, but that's needs a bit of testing work, which is why it hasn't been done yet.